tools/yaml_check.py

```diff
@@ -0,0 +1,28 @@
+import json
+import os
+
+import yaml
+
+
+def yaml_reader(filepath):
+    for file in filepath:
+        with open(file, "rb") as f:
+            data = yaml.safe_load(f)
+            data_str = json.dumps(data)
```

Comment:
This yaml loading a dumping to string seems pointless? You're basically just doing a plain text search in the yaml file, no?

Reply:
No, `safe_load()` is creating a `dict`

Reply:
Not relevant because of this comment: https://github.com/mckinsey/vizro/pull/558#discussion_r1684614728

Reply:
Just to explain my comment in case the changes have done something they weren't meant to: `safe_load` creates a dictionary, but then `json.dumps` turns it straight back into a string and then you scan that string. So we might as well just scan the plain text content of the file without doing the str -> dict -> str conversion, right?